src/dataprotection/setup.py

```diff
@@ -48,7 +48,7 @@
     description='Microsoft Azure Command-Line Tools DataProtectionClient Extension',
     author='Microsoft Corporation',
     author_email='azpycli@microsoft.com',
-    url='https://github.com/Azure/azure-cli-extensions/tree/main/src/dataprotection',
+    url='https://github.com/Azure/azure-cli-extensions/tree/master/src/dataprotection',
```

Comment:
```suggestion
    url='https://github.com/Azure/azure-cli-extensions/tree/main/src/dataprotection',
```

Reply:
This was an autogenerated change. Is this needed to be reverted?

Reply:
ya, since the current one is invalid.